docs/Modules/ecdsa.mdx

```diff
@@ -3,47 +3,44 @@ sidebar_position: 2
 ---
 # ECDSA Ownership Module
 
-This is a basic module that determines "ownership" of the smart account. Smart Accounts are not able to initiate their own transactions on chain and so an Externally Owned Account is needed to sign transactions before they can be sent on chain. The ECDSA ownership module allows us to use a traditional EOA to achieve this. 
+The ECDSA Validation Module is a key component in the Biconomy Modular Smart Account system. It validates user operations signed by an Externally Owned Account (EOA)'s private key. This module enables the EOA to be the owner of the Smart Account (SA) and can be used with other providers like Web3Authn to provide a Web2-like experience for onboarding new users to decentralized applications (dApps).
+[ECDSA ownership module](https://github.com/bcnmy/scw-contracts/blob/main/contracts/smart-account/modules/EcdsaOwnershipRegistryModule.sol) is the most commonly used validator.
```

Comment:
most commonly used / popular "validation module".

Reply:
have removed changes to the module specific pages, since @Aboudjem  has already raised PR for the same